pkg/client/src/app/pages/identities/components/identity-form.tsx

```diff
@@ -29,6 +29,8 @@ import {
 import "./identity-form.css";
 const xmllint = require("xmllint");
 const { XMLValidator } = require("fast-xml-parser");
+const schema = require('./schema.xsd');
```

Comment:
Could we preferably use import instead ?
You might need to add entry to src/typings.d.ts